docs/get-started/tooling/social-login/arcana.mdx

```diff
@@ -3,29 +3,19 @@ title: Arcana
 image: /img/socialCards/arcana.jpg
 ---
 
-[Arcana Network](https://arcana.network) SDKs enable web3 developers to use [social login](https://docs.arcana.network/concepts/social-login) 
-to onboard app users. 
+## Social Login
```

Comment:
We tend to prefer not to start a page with a subheading. Can this be absorbed into the `title` in the front matter? or removed?